index.html

```diff
@@ -0,0 +1,38 @@
+<!-- Edit -->
+
+<!DOCTYPE html>
+<html lang="sv">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello CSS</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Me and CSS</a></li>
+                <li><a href="stack.html">Other Stack</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <h1>Welcome to the world of web-development</h1>
+            <p>From all of us, to all of you, a merry merry:</p> 
+            <p class="consistent">consistent website</p> 
+            <a href="stack.html" class="button">Learn more</a>
+        </section>
+        <section class="content">
+            <h2>Theme of the day</h2>
+            <div class="meme">
+                <p>"Det fungerar på min maskin"</p>
+            </div>
+        </section>
+    </main>
+    <footer id="footer">
```

Comment:
What does this id do? You can refer to the html element directly. I wouldn't really name the id the same as the html tag as the assignment says you have to use the id for a reason and not just because.